test/e2e/index.ts

```diff
@@ -982,6 +985,100 @@ test("can inject agents", async (t) => {
   await cleanSandboxConductors();
 });
 
+test("can query agent meta info over admin and app websocket", async (t) => {
+  const localServices = await runLocalServices();
+  const conductor1 = await launch(
+    ADMIN_PORT,
+    localServices.bootstrapServerUrl,
+    localServices.signalingServerUrl
+  );
+  const {
+    cell_id: cell_id1,
+    client: appClient1,
+    admin: admin1,
+  } = await createAppWsAndInstallApp(ADMIN_PORT);
+
+  await admin1.authorizeSigningCredentials(cell_id1);
+
+  // Retrieve the peer URL of the agent
+  const agentInfos1 = await admin1.agentInfo({ cell_id: null });
+  const agentInfo1 = JSON.parse(agentInfos1[0]).agentInfo;
+  const agentUrl1 = JSON.parse(agentInfo1).url;
+  console.log("agentUrl1: ", agentUrl1);
+
+  // Start a second conductor and install the same app
+  const conductor2 = await launch(
+    ADMIN_PORT_1,
+    localServices.bootstrapServerUrl,
+    localServices.signalingServerUrl
+  );
+
+  const {
+    cell_id: cell_id2,
+    client: appClient2,
+    admin: admin2,
+  } = await createAppWsAndInstallApp(ADMIN_PORT_1);
+
+  await admin2.authorizeSigningCredentials(cell_id2);
+
+  // Now create an entry with agent 1 to get some gossip flowing
+  const acionHash = await appClient1.callZome({
+    cell_id: cell_id1,
+    provenance: cell_id1[1],
+    zome_name: TEST_ZOME_NAME,
+    fn_name: "create_an_entry",
+    payload: null,
+  });
+
+  // Wait until the second agent can get it to make sure that they
+  // have exchanged peer info
+  await retryUntilTimeout<Record>(
+    () =>
+      appClient2.callZome({
+        cell_id: cell_id2,
+        provenance: cell_id2[1],
+        zome_name: TEST_ZOME_NAME,
+        fn_name: "get_an_entry",
+        payload: acionHash,
+      }),
+    null,
+    "agent 2 wasn't able to get the entry of agent 1",
+    200,
+    20_000
+  );
```

Comment:
`dhtSync` here?

Reply:
`dhtSync` is in tryorama so I don't think I can use it (tryorama depends on the js-client so importing tryorama would be a circular dependency).